Using-the-API/API.md

```diff
@@ -716,6 +725,7 @@ ___
 | `reblog`                 | `null` or the reblogged [Status](#status)                                     | yes      |
 | `content`                | Body of the status; this will contain HTML (remote HTML already sanitized)    | no       |
```

Comment:
You might want to mention here that it may now contain shortcodes, which should be replaced with the images in "emojis"

Reply:
Ok. I see. Thank you